source/elements/oneMKL/source/domains/blas/gemm_batch.rst

```diff
@@ -20,12 +24,12 @@ gemm_batch
       **Group API**
 
 
-      .. cpp:function::  void gemm_batch(queue &exec_queue,      buffer<transpose, 1> &transa_array, buffer<transpose,1>      &transb_array, buffer<std::int64_t,1> &m_array,      buffer<std::int64_t,1> &n_array, buffer<std::int64_t,1> &k_array,      buffer<T,1> alpha_array, buffer<T,1> &a_array,      buffer<std::int64_t,1> &lda_array, buffer<T,1> &b_array,      buffer<std::int64_t,1> ldb_array, buffer<T,1> &beta_array,      buffer<T,1> &c, buffer<std::int64_t,1> &ldc_array, std::int64_t      group_count, buffer<std::int64_t,1> &group_size_array)
```

Comment:
cpp is the default, so .. function:: is enough

I have found that .. cpp:function:: does a poor job of formatting multi-line signatures and we should think about just using :: instead so we can have proper line-breaks

Reply:
But will "::" block apply proper cpp syntax highlight?

Reply:
It will do syntax highlighting, but not cross-referencing.  If you use .. parsed-literal::
You can do the cross-referencing manually